syntax = "proto3";

package device;

message DeviceReading {
  int64 arrival_time = 1;
  int64 creation_time = 2;
  string device = 3;
  int32 index = 4;
  string model = 5;
  string user = 6;
  string gt = 7;
  double x = 8;
  double y = 9;
  double z = 10;
}